src/seedu/addressbook/AddressBook.java

```diff
@@ -207,30 +207,29 @@
      */
 
     public static void main(String[] args) {
-        showWelcomeMessage();
-        processProgramArgs(args);
-        loadDataFromStorage();
+        showToUser(DIVIDER, DIVIDER, VERSION, MESSAGE_WELCOME, DIVIDER);
+        if (args.length >= 2) {
+		    showToUser(MESSAGE_INVALID_PROGRAM_ARGS);
+		    exitProgram();
+		}
+		
+		if (args.length == 1) {
```

Comment:
Do note the indentations here, the `if` statements should be at the same level.
Also to make this more readable, you could try using `else if` too.
Also consider using constants instead of 'magic numbers'. 
i.e. `args.length >= 2` can be `args.length >= INVALID_ARGS`